CONTRIBUTING.md

```diff
@@ -62,9 +62,9 @@ If there are code changes in the SDKs, run `changie new` in the corresponding di
 
 Remember to add & commit the release notes fragment.
 This will be used at release time, in the changelog.
-Here is an example of the end-result for all release notes fragments: https://github.com/dagger/dagger/blob/v0.6.4/.changes/v0.6.4.md
+Here is an example of the end-result for all release notes fragments: <https://github.com/dagger/dagger/blob/v0.6.4/.changes/v0.6.4.md>
```

Comment:
Link already work, why add these `<`, `>`? Is it for readability improvement?

Reply:
Forces a URL in Markdown: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links